### ISSUE TITLE
404 URL for submitting a bug report for a package

### ISSUE THREAD

**Issue by crucialfelix** [atom/atom#4933]
The splendid error dialog:

![screen shot 2015-01-09 at 00 07 47](https://cloud.githubusercontent.com/assets/218738/5672746/a95c4062-9793-11e4-9cdd-c3694ddbda3e.png)

links to a 404 URL:

http://git.io/REyP4A

Atom 0.166.0

if this is not in core then just let me know which package to submit this bug for.

**Comment by thedaniel** [comment]
Looks like it's an issue for https://github.com/atom/symbols-view - not sure why it's using a git.io URL, cc @benogle

**Comment by benogle** [comment]
The git.io url is to shorten the url. Windows only allows a 2048 length url. If longer, it wont even open the browser. So all errors create a git.io url with the giantass create issue url.

Not sure what's causing the 404. Maybe git.io failed in this case?

I saw another issue come in about this, but cant find it. 

Does this happen all the time? Or just this once?

**Comment by crucialfelix** [comment]
I tried it 3 times just to be sure, but only with this bug for symbol view.
it may be git.io or the URL shortener. its probably not anything in atom
core.

**Comment by benogle** [comment]
@crucialfelix Can you paste in the URL for that error? When it is visible, run this in the console:

```
document.querySelector('atom-notification.fatal').getIssueUrl()
```

Then paste in the entire URL

**Comment by 50Wliu** [comment]
@benogle here's the other issues: https://github.com/atom/atom/issues/4820#issuecomment-68479529 and #4875.

**Comment by benogle** [comment]
Legit, thanks @50Wliu

**Comment by crucialfelix** [comment]
document.querySelector('atom-notification.fatal').getIssueUrl()
    "https://github.com/atom/atom/issues/new?title=Uncaught%20Error:%20EBADF,%20…0Azentabs,%20v0.6.1%0A%0A%23%20Dev%0ANo%20dev%20packages%0A%60%60%60%0A%0A"

the generated URL was:

    http://git.io/_ycciw

which is an Nginx 404, 
whereas: 

   http://git.io/ycciw

returns a page with "No url found for ycciw"

in this case the error was:
```
fs.js:75
Hide Stack Trace
Error: EBADF, write
  at Error (native)
```

**Comment by benogle** [comment]
The url pasted is the truncated version. When it prints out a long string, you need to click it to get the whole string. Can you click the url string in the console, then paste in the expanded version?

**Comment by crucialfelix** [comment]
I thought it was this at the start of my last comment:

"https://github.com/atom/atom/issues/new?title=Uncaught%20Error:%20EBADF,%20…0Azentabs,%20v0.6.1%0A%0A%23%20Dev%0ANo%20dev%20packages%0A%60%60%60%0A%0A"

that's long, its just escaped, no ? its the result of document.querySelector('atom-notification.fatal').getIssueUrl()

**Comment by benogle** [comment]
Unfortunately, it's still truncated (note the `...`). Please click on the string in the console after running the command, it should expand into a huge string.

**Comment by sedabull** [comment]
I had the exact same issue with an error thrown by package 'terminal-panel' on Ubuntu 14.40, atom v-0.174. I ended up opening an issue manually [here](https://github.com/thedaniel/terminal-panel/issues/24)

**Comment by benogle** [comment]
Example url

```
https://github.com/atom/bracket-matcher/issues/new?title=Uncaught%20TypeError:%20Cannot%20read%20property%20%27getScreenPosition%27%20of%20undefined&body=%5BEnter%20steps%20to%20reproduce%20below:%5D%0A%0A1.%20...%0A2.%20...%0A%0A**Atom%20Version**:%200.170.0%0A**System**:%20Mac%20OS%20X%2010.10.1%0A**Thrown%20From**:%20%5Bbracket-matcher%5D(https://github.com/atom/bracket-matcher)%20package,%20v0.67.0%0A%0A%23%23%23%20Stack%20Trace%0A%0AUncaught%20TypeError:%20Cannot%20read%20property%20%27getScreenPosition%27%20of%20undefined%0A%0A%60%60%60%0AAt%20/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1325%0A%0ATypeError:%20Cannot%20read%20property%20%27getScreenPosition%27%20of%20undefined%0A%20%20at%20TextEditor.module.exports.TextEditor.getCursorScreenPosition%20(/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1325:34)%0A%20%20at%20TextEditor.module.exports.TextEditor.isFoldedAtCursorRow%20(/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:2585:44)%0A%20%20at%20BracketMatcherView.module.exports.BracketMatcherView.updateMatch%20(/Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/bracket-matcher-view.js:111:23)%0A%20%20at%20/Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/bracket-matcher-view.js:94:26%0A%20%20at%20Emitter.module.exports.Emitter.emit%20(/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:82:11)%0A%20%20at%20/Applications/Atom.app/Contents/Resources/app/src/cursor.js:70:25%0A%20%20at%20Emitter.module.exports.Emitter.emit%20(/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:82:11)%0A%20%20at%20Marker.module.exports.Marker.destroyed%20(/Applications/Atom.app/Contents/Resources/app/src/marker.js:264:20)%0A%20%20at%20/Applications/Atom.app/Contents/Resources/app/src/marker.js:51:24%0A%20%20at%20Emitter.module.exports.Emitter.emit%20(/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/node_modules/event-kit/lib/emitter.js:82:11)%0A%20%20at%20Marker.module.exports.Marker.destroy%20(/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/marker.js:318:20)%0A%20%20at%20Marker.module.exports.Marker.destroy%20(/Applications/Atom.app/Contents/Resources/app/src/marker.js:62:25)%0A%20%20at%20Selection.module.exports.Selection.destroy%20(/Applications/Atom.app/Contents/Resources/app/src/selection.js:62:26)%0A%20%20at%20SnippetExpansion.module.exports.SnippetExpansion.setTabStopIndex%20(/Applications/Atom.app/Contents/Resources/app/node_modules/snippets/lib/snippet-expansion.js:120:21)%0A%20%20at%20SnippetExpansion.module.exports.SnippetExpansion.goToNextTabStop%20(/Applications/Atom.app/Contents/Resources/app/node_modules/snippets/lib/snippet-expansion.js:86:18)%0A%20%20at%20Object.module.exports.goToNextTabStop%20(/Applications/Atom.app/Contents/Resources/app/node_modules/snippets/lib/snippets.js:397:43)%0A%20%20at%20atom-text-editor.subscriptions.add.atom.commands.add.snippets:next-tab-stop%20(/Applications/Atom.app/Contents/Resources/app/node_modules/snippets/lib/snippets.js:56:25)%0A%20%20at%20CommandRegistry.module.exports.CommandRegistry.handleCommandEvent%20(/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:243:29)%0A%20%20at%20/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:61%0A%20%20at%20KeymapManager.module.exports.KeymapManager.dispatchCommandEvent%20(/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:558:16)%0A%20%20at%20KeymapManager.module.exports.KeymapManager.handleKeyboardEvent%20(/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:396:22)%0A%20%20at%20HTMLDocument.module.exports.WindowEventHandler.onKeydown%20(/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:167:20)%0A%0A%60%60%60%0A%0A%23%23%23%20Commands%0A%0A%60%60%60%0A%20%20%20%20%20-0:18.2%20core:save%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%20%20%20%20%20-0:16.4%20vim-mode:move-down%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%20%20%20%20%20-0:16.0%20vim-mode:insert-below-with-newline%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%20%20%20%20%20-0:15.6%20editor:newline%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%20%20%20%20%20-0:14.0%20snippets:next-tab-stop%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%20%20%20%20%20-0:14.0%20snippets:expand%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%20%20%20%20%20-0:09.9%20vim-mode:activate-command-mode%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%20%20%20%20%20-0:05.0%20vim-mode:reset-command-mode%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%20%20%20%20%20-0:04.5%20vim-mode:move-down%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%20%20%20%20%20-0:03.9%20vim-mode:activate-insert-mode%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%20%20%20%20%20-0:03.4%20core:move-right%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%20%20%20%20%20-0:03.1%20snippets:next-tab-stop%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%20%20%20%20%20-0:00.0%20window:toggle-dev-tools%20(atom-text-editor.editor.vim-mode.insert-mode)%0A%60%60%60%0A%0A%23%23%23%20Config%0A%0A%60%60%60json%0A%7B%0A%20%20%22core%22:%20%7B%7D%0A%7D%0A%60%60%60%0A%0A%23%23%23%20Installed%20Packages%0A%0A%60%60%60coffee%0A%23%20User%0Aautocomplete-plus,%20v1.1.0%0Avim-mode,%20v0.26.0%0A%0A%23%20Dev%0ANo%20dev%20packages%0A%60%60%60%0A%0A/cc%20@atom/core
```

**Comment by benogle** [comment]
@technoweenie noted that there are occasionally hash collisions when creating new URLs. For now, I will only use the shortener on windows.

**Comment by sedabull** [comment]
@benogle, fyi I just upgraded to v-0.175.0 and this issue seems to be resolved on Ubuntu 14.04

**Comment by benogle** [comment]
Thanks @sedabull. It's intermittent and is an issue on git.io. I'm going to only use the shortener on windows where it's absolutely necessary.
